src/Symfony/Component/Security/Http/Authenticator/AccessTokenAuthenticator.php

```diff
@@ -58,10 +58,11 @@ public function authenticate(Request $request): Passport
         if (!$accessToken) {
             throw new BadCredentialsException('Invalid credentials.');
         }
-        $userIdentifier = $this->accessTokenHandler->getUserIdentifierFrom($accessToken);
+
+        $user = $this->accessTokenHandler->getUserIdentifierFrom($accessToken);
```

Comment:
I would not rename the variable to `$user` as it is never a user anyway. It is either an identifier or a badge

Reply:
I agree, `$identifier` seems the best to me (as @stof said, it's not always a User, it could be a client, an app, a service, or anything else that this requesting the current app)